src/fsharp/SyntaxTree.fsi

```diff
@@ -13,7 +14,10 @@ type Ident =
      new: text: string * range: range -> Ident
      member idText: string
      member idRange: range
-       
+
+/// Represents an identifier with potentially additional trivia information.
+type IdentWithTrivia = Ident * IdentTrivia option
```

Comment:
I think use an actual type for this rather than a tuple.  I think we may want to hide its representation and perhaps later add more information to it. 

Also consider renaming `IdentWithTrivia` --> `SynIdent` and `LongIdentWithTrivia` --> `SynLongIdent`? Then `Ident` becomes more of a compiler-internal TypedTree thing (we can't make it actually internal yet, but I think you see what I mean)